pcdsdevices/epics_motor.py

```diff
@@ -221,6 +186,85 @@ def check_limit_switches(self):
         else:
             return "Low [] High []"
 
+    def get_low_limit(self):
```

Comment:
I prefer the new names for sure - but you didn't _have_ to change them :)
Please also update the tab whitelist with the new names.

Reply:
updated them in b3a7c93 - thanks for pointing this out